config-linux.md

```diff
@@ -270,13 +270,16 @@ For more information, see the kernel cgroups documentation about [memory][cgroup
 **`memory`** (object, OPTIONAL) represents the cgroup subsystem `memory` and it's used to set limits on the container's memory usage.
 For more information, see the kernel cgroups documentation about [memory][cgroup-v1-memory].
 
-The following parameters can be specified to set up the controller:
+Values for memory specify the limit in bytes, or `-1` for unlimited memory.
+
+* **`limit`** *(int64, OPTIONAL)* - sets limit of memory usage
+* **`reservation`** *(int64, OPTIONAL)* - sets soft limit of memory usage
+* **`swap`** *(int64, OPTIONAL)* - sets limit of memory+Swap usage
+* **`kernel`** *(int64, OPTIONAL)* - sets hard limit for kernel memory
+* **`kernelTCP`** *(int64, OPTIONAL)* - sets hard limit for kernel TCP buffer memory
+
+For `swappiness` the values are from `0` (almost never swap) to `100` (very likely to be swapped).
```

Comment:
> I reworded to paraphrase the comment in the code…

But is exposing kernel comments in the spec a good idea?  What if they change the semantics, MUST runtimes translate from the 0–100 range to whatever the local kernel uses?

What's important about this value is that it get written to [`memory.swappiness`][2].  I'd rather leave the config author and host kernel to sort out the `memory.swappiness` semantics between themselves, and have the runtime (and this spec) just blindly pass the opaque value through.

[1]: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git/tree/mm/vmscan.c?h=v4.11.6#n144
[2]: https://www.kernel.org/doc/Documentation/cgroup-v1/memory.txt